### DISCUSSION TITLE
刻意练习

### DISCUSSION THREAD

**Discussion by giscus** [caibingcheng/comment#312] [Announcements]
# 刻意练习

<刻意练习>是一本教人如何练习的书, 是一种从开始如何练习, 到中期如何继续保持提高, 到最后小有所成的练习方法论. 本来很早应该看完, 但是刚好在阅读计划快结束的时候经历了大裁员, 便放下所有事情, 放开心随心玩了一段时间. 而后才开始继续恢复学习状态.

https://imcbc.cn/202302/deliberate-practice/

**Comment by penndu** [comment]
我也看过这本书呢！

**Reply by ryan4yin** [reply]
没看过，但是我之前也写过一篇类似的文章，感觉方法论的书以前看太得多了，现在反而没啥兴趣。
目前我更喜欢依据自己总结出来的一些经验，以及我的兴趣来行动。

**Reply by penndu** [reply]
表示坚持不下去啊！

**Comment by windy177** [comment]
很认同，我认为其实天才论本身就是个骗局，不能否认的是一些人会有来自环境的影响，有了某些领域先知的能力，在某些领域开蒙早。这些是来自老天的眷顾，获取你也有，毕竟每个人的成长的环境不一样，可能你并没有关注到。看了很多身边的人和事情发现往往自律和专注的人容易获得成功，偏执狂往往就是我们理解的“天才”，做一些健康和不舒服的事情，试图让自己变得不一样，是个很富有“多巴胺”的事情。实践 + 总结， 实践+ 总结，水到渠成。
